src/main.rs

```diff
@@ -188,6 +207,8 @@ struct App {
     about_dialog: gtk::AboutDialog,
     app_data_dir: Option<PathBuf>,
     exit_status_code: Arc<Mutex<AppStatusCode>>,
+    #[allow(dead_code)]
```

Comment:
No dead code, please

Reply:
menu object must keepalive, if not, menu disappear after drop

Reply:
done